docs/enroute/Melbourne Centre/BIK.md

```diff
@@ -57,11 +54,28 @@ GUN is reponsible for issuing STAR clearances and initial descent for aircraft i
     *VOZ655 (operated by a SAAB 340) has departed YSCB for YSSY and been handed off from CB TCU to GUN*  
     **VOZ655:** "Melbourne Centre, VOZ655, climbing F130"  
     **GUN:** "VOZ655, Melbourne Centre, cleared ODALE7 arrival, climb F130"
+    
+GUN is also responsible for initial sequencing actions into YSSY. Aircraft cruising above `F250` should be assigned *no lower* than `F250` and handed to BIK for further descent. Aircraft cruising below `F250` should be transferred to BIK at their cruise level.
+
+Refer to the [Sequencing into YSSY](#sequencing-into-yssy) notes below for runway and STAR selection notes.
+
+### Sequencing into YSSY
+Sequencing arrivals from the west into YSSY is a joint responsibility of GUN and BIK. Initial sequencing actions should be performed by GUN, with fine tuning and any holding required issued by BIK.
+
+Aircraft from the south/west are *generally* assigned 16R/34L and aircraft from the north/east assigned 16L/34R to simplify the sequencing required.  However, some heavy aircraft from the north/east may operationally require the longer runway, so consider assigning an aircraft on the ODALE/RIVET STAR 16L/34R to free up a slot on the other runway as necessary. Similarly, it may be beneficial for the sequence to assign an arrival to runway 16L/34R to avoid unnecessary delays for the main runway.
```

Comment:
Same as ARL note. Can we make it say that 16R/34L is to be assigned by default, coord required both ways to assign 16L/34R

Reply:
Done